BioArchLinux/tbtools/PKGBUILD

```diff
@@ -1,35 +1,41 @@
-# Maintainer: kiri <kiri@vern.cc>
+# Maintainer: Kiri <kiri@vern.cc>
 # Contributor: wangzhichao <wangzhichao1990@126.com>
 
 pkgname=tbtools
 _pkgname=TBtools
-pkgver=2023.07.14
+pkgver=2.000
 pkgrel=1
 pkgdesc='GUI/CommandLine Tool Box for biologistists to utilize NGS data. https://doi.org/10.1016/j.molp.2020.06.009'
 arch=('x86_64')
 url='https://github.com/CJ-Chen/TBtools'
-license=('custom')
-depends=('java-runtime' 'bash')
-makedepends=('git' 'unzip')
-source=("git+https://github.com/CJ-Chen/TBtools.git"
+license=('unknown')
```

Comment:
totally wrong

Reply:
how?
https://wiki.archlinux.org/title/PKGBUILD#license
```
If after researching the issue no license can be determined, [PKGBUILD.proto](https://gitlab.archlinux.org/pacman/pacman/blob/master/proto/PKGBUILD.proto) suggests using unknown. However, upstream should be contacted about the conditions under which the software is (and is not) available.
```

Reply:
Let me know if you want to close this PR or not?

Reply:
CJ-Chen just said, available for academic, but he have no license.
I believe the unknow is right.
I don't get your mean, i think i misunderstand u, where  is this bad ?

Reply:
> There are some words from origianl author (you previously mentioned), or this pkg should not be packaged and should be removed!

I don't get this sentence. The 'Original author' mean TBtools's or wiki's ?
If tbtools's, as i said, he do not ban this, we do can re-distribute.
If wiki's, I believe the wiki mean no license as 'unknown', right?

Reply:
OK, i just understand the wiki, you r right. so what should i do? should i write a file to declare CJ-Chen's willing as 'custom' ? or just change to 'custom' and do noting

Reply:
you can echo what he write in English and install it